simulator/src/lib.rs

```diff
@@ -49,14 +50,14 @@
 //!     .build();
 //! let mut window = Window::new("Example", &output_settings);
 //!
-//! egtext!(text = "Hello World!", top_left = Point::zero(), style = text_style!(font = Font6x8, text_color = BinaryColor::On)).draw(&mut display);
+//! Text::new("Hello World!", Point::zero()).into_styled(TextStyleBuilder::new(Font6x8).text_color(BinaryColor::On).build()).draw(&mut display);
```

Comment:
Did you run rustfmt on these? They seem to be pretty long.
```suggestion
//! Text::new("Hello World!", Point::zero()).into_styled(TextStyle::new(Font6x8, BinaryColor::On)).draw(&mut display);
```

Reply:
Hm, the nightly fmt didn't pick this up for some reason. I've formatted it manually.